tasks/install-with-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot.
+  package: name=certbot state=present
```

Comment:
Just had a look one these distributions and added the respective note to the README.

In the meantime, @oxyc commited changes to fix the test setup.